site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,90 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supported storage addons attached to workloads: you need to run 
+`copilot svc deploy` in order to deploy the storage, and the storage is deleted along with the service
+when you run `copilot svc delete`.
+
+Now, you have the option to create environment-level storage addons: the storage is deployed when you run `copilot env deploy`,
+and isn't deleted until you delete the whole environment by running `copilot env delete`.
+
+Similar to the workload-level storage, the environment-level storage is, under the hood, just another [environment addon](../docs/developing/addons/environment.en.md)!
+
+### Best Practice By Default
+Following the best practice in the microservice world, Copilot encourages you to set up storages that are each accessible
+by only one service, instead of monolith storages that are shared by all microservices. Therefore, Copilot assumes
+that your storage is designed to be accessed by one of your services or jobs, even when it is meant to live and die
+with the environment. 
+
+Here is an example of prompts that you might see.
+
+!!! note ""
+	```term
+	$ copilot storage init
+    What type of storage would you like to create?
+    > DynamoDB            (NoSQL)
+      S3                  (Objects)
+      Aurora Serverless   (SQL)
+
+    Which workload needs access to the storage? 
+    > api
+      backend
+
+    What would you like to name this DynamoDB Table? movies
+
+    Do you want the storage to be created and deleted with the api service?
+      Yes, the storage should be created and deleted at the same time as api
+    > No, the storage should be created and deleted at the environment level
+	```
+
+You can skip the prompts using the flags. The following command is equivalent to the prompts above:
+```console
+copilot storage init \
+--storage-type "DynamoDB" \
+--workload "api" \
+--name "movies" \
+--lifecycle "environment"
+```
+
+After you've answered all the prompts or skipped them by flags, Copilot will generate the CloudFormation template that defines the DynamoDB storage
+under your "copilot/environments" directory. In addition, it will generate the necessary access policy; here, one that grants "api" service 
+access to the "movies" storage. The access policy is created as a workload-level addon that lives and dies with your service.
```

Comment:
```suggestion
access to the "movies" storage. The access policy is created as a workload-level addon.
```

Reply:
Changed it to "The access policy is created as a workload addon, meaning that it is deployed and
deleted at the same time as the "api" service."